web/src/api/auth/login-callback.ts

```diff
@@ -95,11 +94,14 @@ export const auth0Login = withApiAuthRequired(
     }
 
     if (user && !user.auth0Id) {
-      // TODO: Sync user's email & name
       try {
-        const userData = await addAuth0Sdk(client).AddAuth0({
+        const userData = await updateUserSdk(client).UpdateUser({
```

Comment:
worth syncing name & email if it changed, even if the user already had an auth0Id

Reply:
Updated